### PR TITLE
Update report.js

### PR DESCRIPTION
**Pull request by dheero**
using let instead of var is now suitable and easy to flexible nowadays.

Fixes #

#### Changes proposed in this pull request:

-change var as let especially in blocks
-use parenthesis and braces in a suitable way for future understanding.
-

### CODE REVIEW

File: interface/forms/track_anything/report.js [hunk 1]
```diff
@@ -14,9 +14,11 @@ function ta_report_getCheckedBoxes(chkboxName) {
   var checkboxes = document.getElementsByName(chkboxName);
   var checkedValue = [];
   // loop over them all
-  for (var i=0; i<checkboxes.length; i++) {
+  for (let i=0; i<checkboxes.length; i++)
```
Comment (adad20) [inline interface/forms/track_anything/report.js]: hi @dheero I think there is no need to provide extra space code format is already according to the standards.

File: interface/forms/track_anything/report.js [hunk 2]
```diff
@@ -47,7 +49,8 @@ function ta_report_plot_graph(formid,ofc_name,the_track_name,ofc_date,ofc_value)
                                      csrf_token_form: csrf_token_js
                                    },
                          dataType: "json",
-                         success: function(returnData){
+                         success: function(returnData)
+                              {
```
Comment (stu01509) [inline interface/forms/track_anything/report.js]: Hello @dheero 

Here should be `success: function(returnData) {`

File: interface/forms/track_anything/report.js [hunk 3]
```diff
@@ -32,8 +34,8 @@ function ta_report_plot_graph(formid,ofc_name,the_track_name,ofc_date,ofc_value)
         //alert("get graph");
         top.restoreSession();
         var checkedBoxes = JSON.stringify(ta_report_getCheckedBoxes("check_col" + formid));
-        var theitems = JSON.stringify(ofc_name);
-        var thetrack = JSON.stringify(the_track_name + " [Track " + formid + "]");
+        let theitems = JSON.stringify(ofc_name);
+        let thetrack = JSON.stringify(the_track_name + " [Track " + formid + "]");
         var thedates = JSON.stringify(ofc_date);
         var thevalues = JSON.stringify(ofc_value);
```
Comment (stu01509) [inline interface/forms/track_anything/report.js]: Here maybe change to let?

**Comment by tywrenn** [comment]
Closed due to inactivity unless anyone has any objections.

**Pull request closed without merging**